README.md

```diff
@@ -53,13 +53,13 @@ As you can see, NewPipe supports multiple video and audio services. Though it st
 
 Partially due to circumstance, and partially due to its popularity, YouTube is the best supported out of these services. If you use or are familiar with any of these other services, please help us improve support for them! We're looking for maintainers for SoundCloud and PeerTube.
 
-If you intend to add a new service, please get in touch with us first! Our [docs](https://teamnewpipe.github.io/documentation/) provide more information on how a new service can be added to the app and to the [NewPipe Extractor](https://github.com/TeamNewPipe/NewPipeExtractor).
+If you intend to add a new service, please get in touch with us first! Our [docs](https://teamnewpipe.github.io/documentation/) provide more information on how a new service can be added to the app and the [NewPipe Extractor](https://github.com/TeamNewPipe/NewPipeExtractor).
 
 ## Description
 
-NewPipe works by fetching the required data from the official API (e.g. PeerTube) of the service you're using. If the official API is restricted (e.g. YouTube) for our purposes, or is proprietary, the app parses the website or uses an internal API instead. This means that you don't need an account on any service to use NewPipe.
+NewPipe works by fetching the required data from the official API (e.g. PeerTube) of the service you're using. If the official API is restricted (e.g. YouTube) for our purposes, or is proprietary, the app parses the website or uses an internal API instead. This means you don't need an account on any service to use NewPipe.
 
-Also, since they are free and open source software, neither the app nor the Extractor use any proprietary libraries or frameworks, such as Google Play Services. This means you can use NewPipe on devices or custom ROMs that do not have Google apps installed.
+Also, since they are free and open source software, neither the app nor the Extractor uses any proprietary libraries or frameworks, such as Google Play Services. This means you can use NewPipe on devices or custom ROMs that do not have Google apps installed.
```

Comment:
Normally, I would agree, but now it breaks number agreement/consistency with "since they are" and that seems wrong. @mhmdanas?

Reply:
The original sentence is a bit awkward too, I would rephrase it like so:

```suggestion
Also, since both are free and open-source software, neither the app nor the Extractor uses any proprietary libraries or frameworks, such as Google Play Services. This means you can use NewPipe on devices or custom ROMs that do not have Google apps installed.
```

(The second change is unrelated to our discussion)

Reply:
@opusforlife2 According to English grammatical rules, this change would actually be correct.

When referring to both the app and the extractor as "they", it is considered plural. This is because they are being referred to in the collective sense. That is easy.

When using the conjunction "neither/nor," the plurality of the verb in the clause depends on the plurality of at least one of the subjects. For the verb to be plural, at least one of the subjects needs to be plural. As both "app" and "extractor" are singular nouns, the verb would be singular.

Shortened version: Two (or more) singular subjects connected by "neither/nor" take a singular verb. If at least one subject is plural, the verb will be plural. Both "app" and "extractor" are singular, so "uses" would be correct.

Source 1: https://archives.cjr.org/language_corner/either_wins.php
Source 2: https://www.myenglishpages.com/english/grammar-lesson-either-or-neither-nor.php
Source 3: I can thank 8th grade English for making me a grammar buff :)

Reply:
Also, I think I prefer using "they" over "both" in this case, as the latter seems a bit awkward.

Better yet, I would suggest using "they are both" or "both of them."

Reply:
I'm okay with "both of them" too. I'll edit my suggestion.

Reply:
@TacoTheDank The sources I checked said exactly what you've said, so that part I was fine with. But the sentence as a whole didn't have number agreement, which I didn't find any precedence for.

I'm okay with the edited suggestion.

Reply:
I don't think the change suggested by the PR is _wrong_ per se, it's just a tad awkward for me (the original sentence is flat-out wrong, as already mentioned).